teerex/sgx-verify/src/collateral.rs

```diff
@@ -203,7 +203,7 @@ pub struct TcbInfo {
 
 impl TcbInfo {
 	/// This extracts the necessary information into the struct that we actually store in the chain
```

Comment:
I think the doc comment should be updated to reflect the change in the return value.

Reply:
I don't understand what you mean by "change in the return value". You refer to the two methods having the same comment?

Reply:
Sorry for that, let me clarify it.
Previously this function returned a single value `TcbInfoOnChain` (e.g. `the struct that we actually store in the chain` ), but now it returns the tuple `(Fmspc, TcbInfoOnChain)`.
I understand that the `Fmspc` will be the key and the `TcbInfoOnChain` will be the value, but the comment saying otherwise could be misleading.
How about this?
```suggestion
	/// This extracts the necessary information into a tuple (`(Key, Value)`) that we actually store in the chain.
```